src/quickgui/qgsquickutils.h

```diff
@@ -74,10 +105,21 @@ class QUICK_EXPORT QgsQuickUtils: public QObject
                                  Qgis::MessageLevel level = Qgis::Warning );
 
     /**
-     * Returns a string with information about screen size and resolution
-     *
-     * Useful to log for debugging of graphical problems on various display sizes
-     */
+      * Get icon from custom theme dir or default if not found in the theme dir
```

Comment:
I think the docs are misleading. Currently it takes the pngs from the library resouces, not possible to use custom dir any more